pkg/process/config/yaml_config.go

```diff
@@ -89,6 +89,10 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 	a.LogLevel = config.Datadog.GetString("log_level")
 	a.StatsdPort = config.Datadog.GetInt("dogstatsd_port")
 
+	if expvarPort := config.Datadog.GetInt(key(spNS, "expvar_port")); expvarPort > 0 {
```

Comment:
Should we add it to the `Known ` params to avoid a warning if it's not set ? (I don't know if warnings are logged for integers)

Reply:
We have another swamp of those now, I'll make another PR to address all of them at once, not just this one.

Reply:
👍